### ISSUE TITLE
I cannot find block 1920000

### ISSUE THREAD

**Issue by shinsy1023** [paritytech/parity#7658]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**: 1.7.12
> - **Which operating system?**: MacOS 
> - **How installed?**: homebrew
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: ethereum classic
> - **Did you try to restart the node?**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

I can't find blocks 1920000, 1920001, 1920002, ...

I ran my parity with command --chain=classic and fully sync

> 2018-01-22 18:12:49  Starting Parity/v1.7.12-stable-f96d152-20180109/x86_64-macos/rustc1.22.1
> 2018-01-22 18:12:49  Keys path /Users/yoonseongmin/Library/Application Support/io.parity.ethereum/keys/Ethereum Classic
> 2018-01-22 18:12:49  DB path /Users/yoonseongmin/Library/Application Support/io.parity.ethereum/chains/classic/db/906a34e69aec8c0d
> 2018-01-22 18:12:49  Path to dapps /Users/yoonseongmin/Library/Application Support/io.parity.ethereum/dapps
> 2018-01-22 18:12:49  State DB configuration: fast
> 2018-01-22 18:12:49  Operating mode: active
> 2018-01-22 18:12:49  Configured for Ethereum Classic using Ethash engine
> 2018-01-22 18:12:50  Updated conversion rate to Ξ1 = US$1069.13 (111349980 wei/gas)
> 2018-01-22 18:12:56  Public node URL: enode://2c15867eb837f0beb29e27fcabda440e7b44de1132466c6a310b77261c10fd7a7339513b43c059311c3a41a379d9b70064898e73763dc3740724f7d50b09b543@192.168.17.237:30303
> 2018-01-22 18:12:59  Imported #5250265 0e18…17f7 (0 txs, 0.00 Mgas, 21.66 ms, 0.52 KiB) + another 3 block(s) containing 21 tx(s)

this is my log when I ran parity

but my code only returns null...

```
var Web3 = require('web3');


if (typeof web3 !== 'undefined') {
  web3 = new Web3(web3.currentProvider);
} else {
  // set the provider you want from Web3.providers
  web3 = new Web3(new Web3.providers.HttpProvider("http://localhost:8545"));
}

web3.eth.getBlock(1920000, function(error, result){ //1968857부터 검색 시작
    if(!error){
    	console.log(web3.eth.blockNumber);
        console.log(result)
    }
    else
        console.error(error);
})
```

**Comment by 5chdn** [comment]
The block will be available once your ancient block sync is finished. Can you confirm you have all ancient blocks available?

**Comment by shinsy1023** [comment]
How can I check it?

**Comment by shinsy1023** [comment]
I deleted my DB with `parity db kill --chain classic`

and I ran it again

> 2018-01-23 10:35:43  Starting Parity/v1.7.12-stable-f96d152-20180109/x86_64-macos/rustc1.22.1
> 2018-01-23 10:35:43  Keys path /Users/yoonseongmin/Library/Application Support/io.parity.ethereum/keys/Ethereum Classic
> 2018-01-23 10:35:43  DB path /Users/yoonseongmin/Library/Application Support/io.parity.ethereum/chains/classic/db/906a34e69aec8c0d
> 2018-01-23 10:35:43  Path to dapps /Users/yoonseongmin/Library/Application Support/io.parity.ethereum/dapps
> 2018-01-23 10:35:43  State DB configuration: fast
> 2018-01-23 10:35:43  Operating mode: active
> 2018-01-23 10:35:43  Configured for Ethereum Classic using Ethash engine
> 2018-01-23 10:35:44  Updated conversion rate to Ξ1 = US$1003.16 (118672616 wei/gas)
> 2018-01-23 10:35:50  Public node URL: enode://2c15867eb837f0beb29e27fcabda440e7b44de1132466c6a310b77261c10fd7a7339513b43c059311c3a41a379d9b70064898e73763dc3740724f7d50b09b543@192.168.17.237:30303
> 2018-01-23 10:35:54  Syncing snapshot 2/178        #0   13/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:35:59  Syncing snapshot 7/178        #0   17/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:36:09  Syncing snapshot 18/178        #0   21/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:36:14  Syncing snapshot 24/178        #0   21/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:36:19  Syncing snapshot 30/178        #0   22/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:36:25  Syncing snapshot 36/178        #0   22/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:36:34  Syncing snapshot 44/178        #0   23/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:36:44  Syncing snapshot 55/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:36:49  Syncing snapshot 61/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:36:59  Syncing snapshot 71/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:37:07  Syncing snapshot 79/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:37:14  Syncing snapshot 86/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:37:19  Syncing snapshot 91/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:37:24  Syncing snapshot 96/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,   0 µs
> 2018-01-23 10:37:37  Syncing snapshot 108/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s, 2380 µs
> 2018-01-23 10:37:44  Syncing snapshot 115/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s, 2380 µs
> 2018-01-23 10:37:54  Syncing snapshot 123/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s, 2380 µs
> 2018-01-23 10:38:05  Syncing snapshot 134/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  2 req/s, 2380 µs
> 2018-01-23 10:38:14  Syncing snapshot 143/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s, 2380 µs
> 2018-01-23 10:38:24  Syncing snapshot 150/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:38:31  Syncing snapshot 156/178        #0   23/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:38:37  Syncing snapshot 162/178        #0   23/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:38:45  Syncing snapshot 172/178        #0   23/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:38:54  Syncing snapshot 175/178        #0   23/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:38:59  Syncing snapshot 177/178        #0   23/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:39:04  Syncing snapshot 177/178        #0   23/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:39:09  Syncing snapshot 177/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:39:14  Syncing snapshot 177/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:39:19  Syncing snapshot 177/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:39:24  Syncing snapshot 177/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:39:34  Syncing snapshot 177/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:39:39  Syncing snapshot 177/178        #0   24/25 peers   8 KiB chain 3 MiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  51 µs
> 2018-01-23 10:39:44  Syncing #5250000 80b9…dcbb     0 blk/s    0 tx/s   0 Mgas/s      0+    0 Qed        #0   24/25 peers   8 KiB chain 1 KiB db 0 bytes queue 10 KiB sync  RPC:  0 conn,  0 req/s,  43 µs


isn't it done?

**Comment by rphmeier** [comment]
the snapshot sync leaves a gap between the genesis and the snapshot block. Your node will download these blocks in the background and they will be unavailable in the meantime

**Comment by shinsy1023** [comment]
thank you very much @rphmeier. how can I know when it finishied?

**Comment by shinsy1023** [comment]
I KNEW IT!

> 2018-01-23 11:39:23    #1731405   24/25 peers   6 MiB chain 71 MiB db 0 bytes queue 17 MiB sync  RPC:  0 conn,  0 req/s,  58 µs

this means 1731405th block has imported!

thanks for your kind

**Comment by 5chdn** [comment]
Exactly :)
